bin/bootstrap

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+
+carthage bootstrap --platform iOS --no-use-binaries
```

Comment:
are we back to having carthage build for us instead of adding the projects to Xcode?